release_tools/semverup.py

```diff
@@ -50,7 +50,13 @@
 @click.command()
 @click.option('--dry-run', is_flag=True,
               help="Do not write a new version number. Print to the standard output instead.")
-def semverup(dry_run):
+@click.option('--bump-major', is_flag=True,
+              help="Increase major version number.")
+@click.option('--bump-minor', is_flag=True,
+              help="Increase minor version number.")
+@click.option('--bump-patch', is_flag=True,
+              help="Increase patch version number.")
```

Comment:
I think it makes more sense to have only an option with three possible values. These values are can not be used at the same time, so it makes sense to have only one. You can use `click` [Choice Options](https://click.palletsprojects.com/en/8.1.x/options/#choice-options) for that.